### ISSUE TITLE
Build fails when upgrading to v3.0.3

### ISSUE THREAD

**Issue by jonnyhork** [muenzpraeger/create-lwc-app#786]
### Summary

We have an Typescript LWC app in a lerna repo and have been locked on `v2.2.1`, we want to upgrade to `v3.0.3` so we can get Webpack 5 and hopefully use some of the latest TS features like optional chaining.

However the build fails with `v3.0.3`.

### Steps To Reproduce:

1.  Change `lwc-services` version package.json to `^3.0.2`
1.  yarn install
2. yarn build

### Expected result

Successful build

### Actual result

Build fails
```bash
@salesforce/soql-builder-ui: $ lwc-services build -w webpack.config.js -m production
@salesforce/soql-builder-ui: ⚡⚡⚡⚡⚡  Lightning Web Components ⚡⚡⚡⚡⚡
@salesforce/soql-builder-ui: ⌛  Creating build configuration
@salesforce/soql-builder-ui: 🌟  Using custom configuration from webpack.config.js.
@salesforce/soql-builder-ui: 📦  Starting build process.
@salesforce/soql-builder-ui: 🆘  The webpack build could not be compiled.
@salesforce/soql-builder-ui: /Users/jhork/Desktop/Workspace/soql-tooling/node_modules/lwc-services/lib/commands/build.js:27
@salesforce/soql-builder-ui:             const { errors } = stats.compilation;
@salesforce/soql-builder-ui:                                      ^
@salesforce/soql-builder-ui: TypeError: Cannot read property 'compilation' of undefined
@salesforce/soql-builder-ui:     at /Users/jhork/Desktop/Workspace/soql-tooling/node_modules/lwc-services/lib/commands/build.js:27:38
@salesforce/soql-builder-ui:     at /Users/jhork/Desktop/Workspace/soql-tooling/node_modules/webpack/lib/webpack.js:138:28
@salesforce/soql-builder-ui:     at processTicksAndRejections (internal/process/task_queues.js:75:11)
@salesforce/soql-builder-ui: error Command failed with exit code 1.
@salesforce/soql-builder-ui: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run build exited 1 in '@salesforce/soql-builder-ui'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Additional information

_Feel free to attach a screenshot or code snippets_.

**Comment by muenzpraeger** [comment]
I'll take a look my AM tomorrow. Can you DM me on Slack a branch of that repo where your change is already in?

**Comment by jonnyhork** [comment]
Yeah sure can, thanks so much!

**Comment by muenzpraeger** [comment]
Reasons for the error was that between Webpack 4 and Webpack 5 the configuration options for webpack changed.

Published with #787 (released as `3.0.4`) a fix that surfaces the Webpack errors better.
